regression/rt_gk_asdex_3x2v_p1.c

```diff
@@ -467,7 +467,6 @@ main(int argc, char **argv)
 
   // field
   struct gkyl_gyrokinetic_field field = {
-    .bmag_fac = ctx.B0, 
```

Comment:
@akashukla there are some 1x mirror input files with this bmag_fac. Delete them please. I believe that model doesn't use it, and we don't want to set a bad example.

Reply:
I think the 1d field model does use these. If I remove them the factor will default to bmag_mid calculated internally. I don't know the mirror tests, so up to you.

Reply:
ah, the boltzmann electron model doesn't use it, so we can delete it in those. But the kinetic electron does, and that's just the polarization_bmag i suggested